clientV2/components/modals/Modal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ModalWrapper = props => {
+  const { heading, children, openModal, closeModal } = props;
```

Comment:
I also think there will be no need for openModal props. 
Doing `this.state.openModal && <ModalWrapper />` will easily solve for the use of openModal.
This will save us extra lines of code.

>   state = {
        openModal: false
      }
>   showModal=()=>{
>    this.setSate({openModal: true})
> }
> 
> closeModal=()=>{
>    this.setSate({openModal: false})
> }
> 
> render(){
>    return(
>       `this.state.openModal && <ModalWrapper`
>         closeModal={this.closeModal}
>         heading="Create Event"
>       `>`
>          `<div> children elements here </div>`
>      `</ModalWrapper>`
>     );}